libs/api-client/src/configuration.ts

```diff
@@ -0,0 +1,132 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Common API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0.0
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+export interface ConfigurationParameters {
+  apiKey?:
+    | string
+    | Promise<string>
+    | ((name: string) => string)
+    | ((name: string) => Promise<string>);
+  username?: string;
+  password?: string;
```

Comment:
Can we configure the generator to exclude these auth options? Feel free to resolve if handling in future PRs/issues.

Reply:
I think we can do that when configuring tRPC adapters, still TBD